api/migrations/20200921100859_change-role-names.js

```diff
@@ -16,16 +16,16 @@ exports.up = async knex => {
     .update({ name: 'eAPD Federal Leadership', isActive: false });
 
   await knex('auth_roles')
-    .where({ name: 'federal' })
-    .update({ name: 'eAPD Federal', isActive: false });
+    .where({ name: 'federal SME' })
+    .update({ name: 'eAPD Federal SME', isActive: false });
 
   await knex('auth_roles')
     .where({ name: 'state coordinator' })
     .update({ name: 'eAPD State Coordinator', isActive: false });
 
   await knex('auth_roles')
-    .where({ name: 'state' })
-    .update({ name: 'eAPD State', isActive: false });
+    .where({ name: 'state SME' })
+    .update({ name: 'eAPD State SME', isActive: false });
```

Comment:
If this migration filename appears in the `knex_migrations` table, this code will not run.

You probably want to generate a new migration and make the necessary changes to the data.

Reply:
I already changed it in the database, the database value is "state SME" and "federal SME" and this migration originally missed it. I changed it so it would work in prod.